### DISCUSSION TITLE
~/.config/i3/config

### DISCUSSION THREAD

**Discussion by WgpArch** [i3/i3#5460] [Q&A]
I add at the bottom of ~/.configi3/config  exec_always --no-startup-id feh --bg-fill /path/to/wallpaper.jpg for my wallpaper but it does not work and mouse curser only turns in circles. I try this  exec_always nitrogen --no-startup-id --restore it too does not work and again mouse curser tuns in circles,but manually nitrogen puts it there. Then for my conky I do exec conky it comes up after re login but windows does not go over it instead they go behind it. Is there something I do wrong or is there things I must change?

**Comment by orestisfl** [comment]
What happens when you run these commands on the terminal? Do they work as expected?

If yes, when you configure them to run via your i3 config you might want to redirect their output to a file so you can inspect their logs.

**Reply by WgpArch** [reply]
I got my wallpaper going but not conky
exec nitorgen --restore did it. Still trying some stuff for conky.

**Reply by WgpArch** [reply]
Ok conky comes up nut all wondows stiil go to the edge of conky and behind,I tried chinging thing im my conky script. To no avail I will go that for now till I can figure something out thanks for your time.

**Reply by deanr72** [reply]
My line looks slightly different in order to yours. Feel free to try it:

exec --no-startup-id feh ~/Pictures/Wallpapers/* --recursive --randomize --bg-fill

**Reply by WgpArch** [reply]
thanks will try it out I am just having a hard time with conky's cause I tried different scripts but all my window go behind the conky not over it. I tried own window override own window class conky own window panel but everytime it does not work,but on my openbox config it works 100%
